sites/website/docs/api/fast-element.attributedefinition._constructor_.md

```diff
@@ -21,8 +21,8 @@ constructor(Owner: Function, name: string, attribute?: string, mode?: AttributeM
 
 |  Parameter | Type | Description |
 |  --- | --- | --- |
-|  Owner | <code>Function</code> |  |
-|  name | <code>string</code> |  |
-|  attribute | <code>string</code> |  |
-|  mode | <code>AttributeMode</code> |  |
-|  converter | <code>ValueConverter &#124; undefined</code> |  |
+|  Owner | Function |  |
```

Comment:
Are all of these removals of backticks and <code> a stylistic choice or have they been done through some automated process?